bower.json

```diff
@@ -1,15 +1,19 @@
 {
   "name": "virtual-each",
   "dependencies": {
-    "ember": "1.13.7",
+    "ember": "components/ember#release",
```

Comment:
Any reason for this change? I think we'd like to keep the min ember dep at 1.13.x